app/src/main/java/asd/org/ahorcado/models/AbstractMatch.java

```diff
@@ -53,4 +53,8 @@ public boolean afterExecute(char letter) throws MatchLostException {
     public String getNewWord() {
         return guessWord.getNewWord();
     }
+
+    public void initialGame(){
+        setLife(6);
```

Comment:
Could you put this hardcoded "6" in a constant? Or xml config? wherever you want.